mappings/net/minecraft/client/gui/screen/ingame/HandledScreen.mapping

```diff
@@ -62,13 +65,21 @@ CLASS net/minecraft/class_465 net/minecraft/client/gui/screen/ingame/HandledScre
 		ARG 1 keyCode
 		ARG 2 scanCode
 	METHOD method_2385 drawSlot (Lnet/minecraft/class_4587;Lnet/minecraft/class_1735;)V
```

Comment:
Just need to ensure the draw vs render consistency. Why draw than render here?

Reply:
I think I'll revert the `renderBackground/Foreground` renames for now because I don't feel like standardising it across all screens (someone should still do that, though).